frontend/src/service/SessionService.ts

```diff
@@ -0,0 +1,31 @@
+import { UserCredentials, InvalidCredentialsError } from "@/types/UserCredentials";
+import { Session } from "@/types/Session";
+import axios, { AxiosResponse } from "axios";
+
+const CONFIG = {
+    baseURL: "API AUTH SERVER URL" + "/login", //Set from .env
```

Comment:
Think this should be resolved in this PR, so it's not forgotten about.
Unless of course this is blocked by something else, in which case resolve this comment yourself.

Reply:
Referring to the `//Set from .env` comment